src/rviz/ogre_helpers/render_widget.h

```diff
@@ -62,6 +62,8 @@ class RenderWidget: public QWidget
   Ogre::RenderWindow* render_window_;
 
   QFrame* renderFrame;
+
+  int pixel_ratio_;
```

Comment:
I would initialize the variable to 1 here, then you can remove the assignment when Qt < 5 and you are always safe if someone does something wrong with the pre-processor macros.

Reply:
Please keep the variable double/float. AFAIK, pixel ratio does not have to be integers. For example, setting "export QT_SCALE_FACTOR=2.5" makes QWindow::devicePixelRatio() returns 2.5.

Reply:
Fixed.